product_listings_manager/config.py

```diff
@@ -1,5 +1,29 @@
-# composedb postgres settings
-DBNAME = "compose"
-DBHOST = "db.example.com"
-DBUSER = "myuser"
-DBPASSWD = "mypassword"
+"""PLM configuration module."""
+import os
+
+
+class Config(object):
+    """Default config."""
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    SQLALCHEMY_ECHO = False
+    SQLALCHEMY_DATABASE_URI = 'postgresql://user:pass@localhost/compose'
+
+
+class DevConfig(Config):
+    """Development config."""
+    SQLALCHEMY_TRACK_MODIFICATIONS = True
```

Comment:
What's the point? PLM does not do any write operations, so it should not modify the returned objects.

Reply:
It's not needed in PLM. Deleted.